cards/src/main/resources/cards/custom/group3/bug/verdant_dreams/spell_dreams_of_strength.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Dreams of Strength",
+  "baseManaCost": 7,
+  "type": "SPELL",
+  "heroClass": "AMBER",
+  "rarity": "EPIC",
+  "description": "Give a Larva +8/+8 and Taunt.",
```

Comment:
this might also be way too strong, seeing as amber has no problem generating larva

Reply:
Makes sense. It'll be moved to +7/+7 and taunt for the same mana cost. This essentially is the same as an Ironbark Protector (1 mana 1/1 with +7/+7 and Guard), which I think is more balanced.